tests/test_flow_decomposition.py

```diff
@@ -0,0 +1,27 @@
+#
+# Copyright (c) 2020-2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import pathlib
+import pandas as pd 
+
+import pypowsybl as pp
+import pypowsybl.flowdecomposition as fd
+
+TEST_DIR = pathlib.Path(__file__).parent
+DATA_DIR = TEST_DIR.parent / 'data'
+
+def test_run_fd():
+    net = pp.network.load(str(DATA_DIR / 'NETWORK_PST_FLOW_WITH_COUNTRIES_NON_NEUTRAL.uct'))
```

Comment:
you can use DATA_DIR.joinpath('NETWORK_PST_FLOW_WITH_COUNTRIES_NON_NEUTRAL.uct')

Reply:
fixed